packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -349,9 +381,24 @@ export class PersonalDetailsForm extends React.Component {
             form_initial_values.address_state = '';
         }
         if (this.props.is_mf) {
-            form_initial_values.tax_residence = form_initial_values.tax_residence
-                ? getLocation(residence_list, form_initial_values.tax_residence, 'text')
-                : '';
+            if (form_initial_values.tax_residence) {
+                const is_single_tax_value = form_initial_values.tax_residence.indexOf(',') < 0;
+                if (is_single_tax_value) {
+                    form_initial_values.tax_residence = getLocation(
+                        residence_list,
+                        form_initial_values.tax_residence,
+                        'text'
+                    );
+                } else {
+                    const tax_residences = [];
+                    form_initial_values.tax_residence.split(',').forEach(residence => {
+                        tax_residences.push(getLocation(residence_list, residence, 'text'));
```

Comment:
if it is in a string, i think we should trim residence `residence.trim()`

Reply:
it comes from  API in this format `de,pl`